pkg/cli/verifier/control_plane.go

```diff
@@ -48,9 +47,9 @@ func NewControlPlaneHealthVerifier(stdout io.Writer, stderr io.Writer, kubeClien
 		bootstrapProber: &podProber{
 			kubeClient: kubeClient,
 			restConfig: restConfig,
-			port:       crdconversion.HealthzPort,
+			port:       constants.CRDConversionWebhookPort,
 			path:       constants.WebhookHealthPath,
-			protocol:   constants.ProtocolHTTP,
+			protocol:   constants.ProtocolHTTPS,
```

Comment:
I'm not 100% sure what this is being used for, but OSM doesn't support protocol HTTPS

Reply:
health check and metrics do